packages/service-worker-mock/models/Response.js

```diff
@@ -1,7 +1,16 @@
 // stubs https://developer.mozilla.org/en-US/docs/Web/API/Response
-class Response {
+const Body = require('./Body');
+const Blob = require('./Blob');
+
+class Response extends Body {
   constructor(body, init) {
-    this.body = body || '';
+    if (typeof body === 'object' && !(body instanceof Blob)) {
+      body = JSON.stringify(body);
+    } else if (typeof body !== 'object') {
+      body = undefined;
+    }
+    super(body);
```

Comment:
Do we need the if statements? I think it should just be
```js
constructor(body, init) {
  super(body);
  ...
}
```

Reply:
When you clone the Response the body will be already a Blob